### DISCUSSION TITLE
No internet connection during installation

### DISCUSSION THREAD

**Discussion by lorenzoiuri** [SkyrilHD/Dell-Exx50-Hackintosh#48] [Q&A] [answered]
Hello, I'm very new to OpenCore and I'm encountering difficulties during the installation.
I have a latitude E7450.
My wireless card (AC 7265) and Gigabit card are not available during the installation, despite having read that they should be supported.
I have read also that the kext are disabled by default but in the config.plist file I see both kext enabled.
Am I missing something?

Also, is there a way to configure the networking during install (like a network manager)?

Thanks! :)

**Comment by SkyrilHD** [comment]
Which OS version are you trying to install? The latest EFI "only" supports Monterey. For older versions, you need to swap the AirportItlwm kext with the suitable version. The LAN port is definitely working for me. Is it also Intel? (Intel I218LM3)

**Reply by lorenzoiuri** [reply]
I'm trying to install Monterey, 
These are the models of the network interfaces.
![image](https://user-images.githubusercontent.com/1856869/162559327-f46318ad-e4ec-4b53-8b67-7632e5f9fa8e.png)
![image](https://user-images.githubusercontent.com/1856869/162559341-bf86773d-9f00-4100-bf7f-33f4538c8a90.png)

Is the EFI folder provided in this repo self-contained or do I need other files?

Thanks again for your support

**Reply by lorenzoiuri** [reply]
I'm also attaching the dmesg e ifconfig logs.

[dmesg.txt](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/files/8456407/dmesg.txt)
[ifconfig.txt](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/files/8456408/ifconfig.txt)

**Comment by lorenzoiuri** [comment]
Booting the installer with the ethernet cable connected I was able to perform the installation.
Wifi still is not working though. Are there logs I can give to help the troubleshooting?

**Comment by lorenzoiuri** [comment, marked as answer]
Ok, I was really missing something.
In the config.plist file the option Enabled was set to false.
Enabling it made the wifi working!
